runner/execer/os/os_execer.go

```diff
@@ -577,5 +597,52 @@ func cleanupProcs(pgid int) (err error) {
 				"error": err,
 			}).Error("Error cleaning up pgid")
 	}
+
 	return err
 }
+
+// getUserCurrentMemUsage compute the memory (rss) used by all processes owned by the current user
+func getUserMemUsage(pg procGetter) (int, error) {
+	pm, err := pg.getProcs()
+	if err != nil {
+		return 0, err
+	}
+	cUser, err := user.Current()
+	if err != nil {
+		return 0, err
+	}
+
+	// compute the total user's memory usage
+	totalMem := 0
+	for _, proc := range pm.byUser[cUser.Username] {
```

Comment:
Do we know how this will work e.g. on a shared machine with 2 separate scoot workers running?

Reply:
good question - I don't really know.

Reply:
I was able to verify that the ps command results included processes from multiple scoot workers running on the same machine.... I didn't see a simple way to restrict the process list to those for a specific scoot worker, so I updated the code to only report memory usage when it can find only 1 './workerserver' process.  

Also, this code only works on linux machines. It fails on mac because it uses ps options (user, cmd) that are not available on macs.  The code could be refactored to be platform specific, but I'm not sure it's worth it.

I will not merge this branch.